src/app/component/dashboard/marketing/massive-email/massive-email.component.ts

```diff
@@ -37,10 +50,63 @@ export class MassiveEmailComponent implements OnInit {
       .getConfiguration("administarator", "massive-email")
       .subscribe((config) => {
         this.configField = config;
+        this.getEmailDrafts();
+
         this.loading = false;
       });
   }
 
+  getEmailDrafts(selectNewlyCreated?: boolean) {
```

Comment:
All these new functions also repeat two times(for email and SMS). It looks pretty the same.
It's problem for future development and maintenance. If we need to change something or add, we need to do it on a few places.
If the same code repeats a few times it does not need to be copy-paste into different components.
It's needed to make reusable components (or services).